<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_null_warnOutDialog</class>
 <widget class="QDialog" name="dialog_null_warnOutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="checkBox_warnOutDialog_sendToOther">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送至分控</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_warnOutDialog_emailWarn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>邮件报警</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_warnOutDialog_openVideo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开启录像</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_warnOutDialog_sendPic">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送截图</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_warnOutDialog_capturePic">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>抓拍图像</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_warnOutDialog_callPreset">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>调用预置点</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_warnOutDialog_openWarnOut">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开启报警输出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_warnOutDialog_continuTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>报警持续时间</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_warnOutDialog_continueTime">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_warnOutDialog_warn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>蜂鸣器报警</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_warnOutDialog_sendToOther">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>视频通道</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_warnOutDialog_sendToOther">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_warnOutDialog_sendPic">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>视频通道</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_warnOutDialog_sendPic">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_warnOutDialog_openVideo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>视频通道</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_warnOutDialog_openVideo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_warnOutDialog_callPreset">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>视频通道</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_warnOutDialog_callPreset">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_warnOutDialog_capturePic">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>视频通道</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_warnOutDialog_capturePic">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_warnOutDialog_id">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>67</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>位置号</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_warnOutDialog_id">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_warnOutDialog_use">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>应用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_warnOutDialog_sure">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_warnOutDialog_cancel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>330</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_warnOutDialog_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>报警输出</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_warnOutDialog_shutdown">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>0</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
